packages/ui/src/index.ts

```diff
@@ -1,6 +1,57 @@
+// import { FontFamilyTokens } from "@tamagui/web";
+import { FontFamilyTokens } from "@tamagui/core";
+import {
+  getTokenValue,
+  SizeTokens,
+  SpaceTokens,
+  Token,
+  getConfig,
+  Variable,
+} from "tamagui";
+
+export { useTheme } from "tamagui";
 export * from "./components/Button";
+export * from "./components/Icon";
+export * from "./components/ListItem";
 export * from "./components/TamaguiProvider";
+export * from "./components/Text";
+export * from "./components/View";
 export * from "./tamagui.config";
-export * from "./components/Icon";
 export * from "./components/UrbitSigil";
-export { ZStack, View } from "tamagui";
+
+export const token = {
```

Comment:
I didn't end up using these helpers -- I'd remove, I'm not sure this is how we'd want them shaped anyway.

Reply:
(Referring to `token`, `getFont`, and `getFontTokenValue`.